src/app/tables/table.component.css

```diff
@@ -58,20 +57,14 @@
         margin-bottom: 3rem;
     }
 
-    .estuary-table-header {
-        /* Make the width 100% minus the padding used by div.estuary-container */
-        width: calc(100% - 10rem);
+    .estuary-inner-container {
+        max-width: 350px;
+        min-width: 10px;
```

Comment:
Can you set the min-width to 640rem or pixels. That will keep things from getting squished if you only have one small column activated.